master/internal/experiment/bulk_action.go

```diff
@@ -798,6 +798,20 @@ func MoveExperiments(
 			return nil, fmt.Errorf("updating run's project IDs: %w", err)
 		}
 
+		if _, err = tx.NewRaw(`UPDATE runs SET local_id=s.local_id FROM runs as r 
+		JOIN (SELECT r.id as id, (p.max_local_id + ROW_NUMBER() OVER(PARTITION BY p.id)) as local_id
+		FROM projects p JOIN runs r ON r.project_id=p.id) as s ON r.id=s.id WHERE r.experiment_id IN (?)`,
+			bun.In(validIDs)).Exec(ctx); err != nil {
+			return nil, fmt.Errorf("updating run's local IDs: %w", err)
+		}
+
+		if _, err = tx.NewRaw(`UPDATE projects SET max_local_id=s.max_local_id FROM projects as p
+		JOIN (SELECT project_id, COALESCE(MAX(local_id), 1) as max_local_id FROM runs GROUP BY project_id
+		ORDER BY project_id) as s ON p.id=s.project_id WHERE p.id=?`,
```

Comment:
Same nesting/indentation nit above; if you don't feel like that makes it more readable, no pressure to change it.

Also, why have an `ORDER BY` clause in the subquery? Shouldn't hurt anything (other than perhaps slowing down the query), but also doesn't seem like it'd help.

Reply:
Fixed. Thanks, the order by was unnecessary.